client/src/App.scss

```diff
@@ -73,3 +73,22 @@ homepage-map {
     font-weight: bold;
     font-size: 24px;
 }
+
+.mobile-search-modal {
```

Comment:
This stuff needs to be in App.scss because the modal component puts its elements high up in the element tree, above where the Header.scss can target